### PR TITLE
Created Digigrade Legs Tool

### PR DESCRIPTION
**Pull request by 989onan**
Will need a translation for "ManualPanel.CreateDigigradeLegs.label"

### CODE REVIEW

File: ui/manual.py [hunk 1]
```diff
@@ -126,6 +126,12 @@ def draw(self, context):
             row = col.row(align=True)
             row.scale_y = button_height
             row.operator(Armature_manual.OptimizeStaticShapekeys.bl_idname, icon='MESH_DATA')
+            
+            if not Common.version_2_79_or_older():
+                col.separator()
```
Comment (feilen) [inline ui/manual.py:131]: Suggest removing this, it just adds a gap in the panel

File: tools/armature_manual.py [hunk 2]
```diff
@@ -1376,6 +1376,129 @@ def execute(self, context):
...
+
+    cb.head = b.head
+    cb.tail = b.tail
+    cb.matrix = b.matrix
+    cb.parent = b.parent
+    return cb
+
+@register_wrap
+class CreateDigigradeLegs(bpy.types.Operator):
+    """Create Digigrade Legs While In Edit Mode With Digigrade Thighs Selected."""
```
Comment (feilen) [inline tools/armature_manual.py:1392]: Typo, 'Digitigrade Legs'. No need for title case here

File: tools/armature_manual.py [hunk 3]
```diff
@@ -1376,6 +1376,129 @@ def execute(self, context):
...
+    bl_idname = "armature.createdigigradelegs"
+    bl_label = "Create Digi-grade Legs"
```
Comment (feilen) [inline tools/armature_manual.py:1394]: Same here

File: tools/armature_manual.py (hunk 2 above)
Comment (feilen) [reply tools/armature_manual.py:1392]: (no need to change the code imo, just ui)

File: ui/manual.py [hunk 4]
```diff
@@ -126,6 +126,12 @@ def draw(self, context):
             row = col.row(align=True)
             row.scale_y = button_height
             row.operator(Armature_manual.OptimizeStaticShapekeys.bl_idname, icon='MESH_DATA')
+            
+            if not Common.version_2_79_or_older():
```
Comment (feilen) [inline ui/manual.py:130]: We should probably add a 'how to use' button like we did for Bake (just link to the FA page I guess) that explains how to use it, but we can deal with all that after the merge

**Review by feilen — CHANGES_REQUESTED** [review]
Only a couple little things to change. Looks great!

**Comment by 989onan** [comment]
Changes made via commit

**Review by feilen — APPROVED** [review]

**Pull request merged**